src/coreclr/src/debug/createdump/main.cpp

```diff
@@ -5,14 +5,14 @@
 
 #ifdef HOST_WINDOWS
 #define DEFAULT_DUMP_PATH "%TEMP%\\"
-#define DEFAULT_DUMP_TEMPLATE "dump.%d.dmp"
+#define DEFAULT_DUMP_TEMPLATE "dump.%p.dmp"
 #else
 #define DEFAULT_DUMP_PATH "/tmp/"
-#define DEFAULT_DUMP_TEMPLATE "coredump.%d"
+#define DEFAULT_DUMP_TEMPLATE "coredump.%p"
 #endif
 
 const char* g_help = "createdump [options] pid\n"
-"-f, --name - dump path and file name. The pid can be placed in the name with %d. The default is '" DEFAULT_DUMP_PATH DEFAULT_DUMP_TEMPLATE "'\n"
+"-f, --name - dump path and file name. The %p, %e, %h %t format characters are supported. The default is '" DEFAULT_DUMP_PATH DEFAULT_DUMP_TEMPLATE "'\n"
```

Comment:
AFAIK, these are common in Linux, but not on Windows. Maybe you should add a note on their usage here on top of the design doc.

Also, changing an existing option might be a breaking change. `%d` already exists on Linux, so I understand the consistency part, but we already have users and not having the option here is not great (I did see that it's going to work nonetheless). See for example in [aspnetcore](https://github.com/dotnet/aspnetcore/blob/ef1f48ab97174131f1753efe9241295b1309d3e2/eng/helix/content/RunTests/ProcessUtil.cs#L103)

Reply:
I did add a section to the design doc about this [here](https://github.com/mikem8361/runtime/blob/fixdumps/docs/design/coreclr/botr/xplat-minidump-generation.md#configurationpolicy). You will need to scroll down a little.  

As you said %d is still supported but I want to de-emphasize this option.

Reply:
Then can we link to it. My problem is around information discoverability.

Reply:
You mean link to the design doc section in the createdump help message?  I can see display the options with there descriptions in the help maybe.

Reply:
What about this for the help message:

```
createdump [options] pid
-f, --name - dump path and file name. The default is '%TEMP%\dump.%p.dmp'. These specifiers are substituted with following values:
   %p  PID of dumped process.
   %e  The process executable filename.
   %h  Hostname return by gethostname().
   %t  Time of dump, expressed as seconds since the Epoch, 1970-01-01 00:00:00 +0000 (UTC).
-n, --normal - create minidump.
-h, --withheap - create minidump with heap (default).
-t, --triage - create triage minidump.
-u, --full - create full core dump.
-d, --diag - enable diagnostic messages.
```